pkg/entity/register/worker.go

```diff
@@ -112,7 +112,14 @@ func (w *worker) send(ctx context.Context, batch map[entity.Key]fwrequest.Entity
 
 	var entities []entity.Fields
 	for _, r := range batch {
-		entities = append(entities, r.Data.Entity)
+		entity := r.Data.Entity
+		// Add labels to Metadata
+		if r.Definition.Labels != nil && len(r.Definition.Labels) > 0 {
```

Comment:
Is it required to check the `nilness`? If `Labels` is `nil`, `len(r.Definition.Labels)` will be 0, won't it?